client/src/modules/dashboard/Calendar/pages/Calendar.tsx

```diff
@@ -1,64 +1,95 @@
-import React, { ReactElement } from 'react';
+import { Button, Flex, Heading } from '@chakra-ui/react';
+import { DataTable } from 'chakra-data-table';
 import NextError from 'next/error';
-import { Flex, Heading } from '@chakra-ui/layout';
-import { Button } from '@chakra-ui/button';
+import React, { ReactElement } from 'react';
+
+import {
+  useCalendarIntegrationStatusQuery,
+  useTokenStatusesQuery,
+} from '../../../../generated/graphql';
 import { checkPermission } from '../../../../util/check-permission';
 import { Permission } from '../../../../../../common/permissions';
 import { useAuth } from '../../../auth/store';
 import { Layout } from '../../shared/components/Layout';
+import { DashboardLoading } from '../../../../modules/dashboard/shared/components/DashboardLoading';
 import { NextPageWithLayout } from '../../../../pages/_app';
 
 const serverUrl = process.env.NEXT_PUBLIC_SERVER_URL || 'http://localhost:5000';
 
 export const Calendar: NextPageWithLayout = () => {
-  const { user } = useAuth();
+  const { user, isLoggedIn } = useAuth();
```

Comment:
```suggestion
  const { user } = useAuth();
```
It doesn't look like we need this